src/app/code/community/Shopgate/Cloudapi/etc/config.xml

```diff
@@ -128,4 +128,16 @@
             </shopgate_cloudapi_delete_expired_tokens>
         </jobs>
     </crontab>
+    <default>
+        <shopgate_cloudapi>
+            <layout>
+                <styles>
+                    .page-title {display: none;}
```

Comment:
just to be safe, lets wrap it with CDATA, have a feeling where one developer will come in one day and plop an illegal xml character in there that is also a valid CSS character., e.g. child selector ```>```
```
<![CDATA[
]]>
```

Reply:
I also worry about the implementation of the default values. What I see as a problem is that if a customer says, "oh, I want to delete another element". So they go to the Sys>Config and add css to remove an element. Bam, they have a "back" link and other elements which are removed by default showing up on the page. Ya know?

Reply:
Added the cdata. I also second your concerns. This is justthe first implementation. Second one could inject the config using update script. We decided to not do this as the first approach in order to avoid messing around with further update scripts just for minor css changes. If we have a solid default css, we can think about a more robust solution.